release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -101,6 +101,27 @@ type of private DNS zone.
 // Monitor whether this info moves to a KB article or remains in official docs.
 // If it moves to KB, permanently remove this entry.
 
+[id="ocp-4-5-openstack-custom-networking"]
+==== Custom networks and subnets for clusters on {rh-openstack}
+
+{product-title} {product-version} introduces support for installing clusters on {rh-openstack-first}
+ that rely on preexisting networks and subnets.
+
+[id="ocp-4-5-openstack-additional-networks"]
+==== Additional networks for clusters on {rh-openstack}
+
+{product-title} {product-version} introduces support for multiple networks in clusters that run on {rh-openstack}.
+You can specify these networks for both control plane and compute machines during installation.
+
+[id="ocp-4-5-openstack-kuryr-octavia"]
+==== Improved {rh-openstack} load balancer upgrade experience for clusters that use Kuryr
+
+Previously, clusters that were installed on {rh-openstack} version 13 and then upgraded to {rh-openstack} version 16 did not take full advantage of {rh-openstack}'s load balancing features.
+
+Upgraded clusters that use Kuryr now have improved support for Octavia load balancing services, including the Octavia OVN provider driver.
```

Comment:
This should be:
Clustter that use Kuryr now have improved support for Octavia load balancing services on {rh-openstack} clusters that were updated from 13 to 16, including the Octavia OVN provider driver